charts/flink-job/values.yaml

```
@@ -26,6 +26,20 @@ imagePullSecrets: []
 # -- Which Flink version to use
 version: v1_16
 
+internalSsl:
+  # -- Set up SSL authentication/encryption using an init-container for creating the certificate
+  enabled: true
+  configuration:
+    keystore: /flinkkeystore/truststore.jks
+  podVolumes:
+    - name: truststore
+      secret:
+        secretName: truststore.jks
+  podVolumeMounts:
```

Volume mounts are container-specific, and not pod-specific
```suggestion
  volumeMounts:
```